.github/workflows/deploy_lambda.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy Lambda
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Actually can we set this only to deploy if there are changes to the lambda itself?

Reply:
Makes sense, I'll do that

Reply:
Similar to:

https://github.com/pytorch/builder/blob/da1928b8e919c362ada44ef5a9b2e10bc7f518f8/.github/workflows/build-conda-images.yml#L3-L7

```
on:
  pull_request:
    paths:
      - conda/Dockerfile
      - 'common/*'
 ```